core/src/main/scala/org/broadinstitute/dsde/rawls/webservice/MethodConfigApiService.scala

```diff
@@ -24,106 +24,113 @@ trait MethodConfigApiService extends HttpService with PerRequestCreator with Use
-              perRequest(requestContext, WorkspaceService.props(workspaceServiceConstructor, userInfo),
-                WorkspaceService.CreateMethodConfiguration(WorkspaceName(workspaceNamespace, workspaceName), methodConfiguration))
-          }
-        }
-    } ~
-      path("workspaces" / Segment / Segment / "methodconfigs" / Segment / Segment) { (workspaceNamespace, workspaceName, methodConfigurationNamespace, methodConfigName) =>
-        get {
-          requestContext =>
-            perRequest(requestContext, WorkspaceService.props(workspaceServiceConstructor, userInfo),
-              WorkspaceService.GetMethodConfiguration(WorkspaceName(workspaceNamespace, workspaceName), methodConfigurationNamespace, methodConfigName))
-        } ~
-          put {
-            entity(as[MethodConfiguration]) { newMethodConfiguration =>
-              requestContext => {
-                perRequest(requestContext, WorkspaceService.props(workspaceServiceConstructor, userInfo),
-                  WorkspaceService.OverwriteMethodConfiguration(WorkspaceName(workspaceNamespace, workspaceName), methodConfigurationNamespace, methodConfigName, newMethodConfiguration))
-              }
-            }
-          } ~
-          post {
-            entity(as[MethodConfiguration]) { newMethodConfiguration =>
-              requestContext => {
-                perRequest(requestContext, WorkspaceService.props(workspaceServiceConstructor, userInfo),
-                  WorkspaceService.UpdateMethodConfiguration(WorkspaceName(workspaceNamespace, workspaceName), methodConfigurationNamespace, methodConfigName, newMethodConfiguration))
-              }
-            }
-          } ~
-          delete {
-            requestContext =>
-              perRequest(requestContext, WorkspaceService.props(workspaceServiceConstructor, userInfo),
-                WorkspaceService.DeleteMethodConfiguration(WorkspaceName(workspaceNamespace, workspaceName), methodConfigurationNamespace, methodConfigName))
-          }
+            WorkspaceService.ListAgoraMethodConfigurations(WorkspaceName(workspaceNamespace, workspaceName)))
       } ~
-      path("workspaces" / Segment / Segment / "methodconfigs" / Segment / Segment / "validate") { (workspaceNamespace, workspaceName, methodConfigurationNamespace, methodConfigName) =>
-        get {
+      post {
+        entity(as[MethodConfiguration]) { methodConfiguration =>
           requestContext =>
             perRequest(requestContext, WorkspaceService.props(workspaceServiceConstructor, userInfo),
-              WorkspaceService.GetAndValidateMethodConfiguration(WorkspaceName(workspaceNamespace, workspaceName), methodConfigurationNamespace, methodConfigName))
+              WorkspaceService.CreateMethodConfiguration(WorkspaceName(workspaceNamespace, workspaceName), methodConfiguration))
         }
+      }
+    } ~
+    path("workspaces" / "v2" / Segment / Segment /  "methodconfigs") { (workspaceNamespace, workspaceName) =>
```

Comment:
We could also call this something like
```
"workspaces" / Segment / Segment /  "allRepoMethodConfigs"
```
which may be more comprehensible to API users.